requirements.in

```diff
@@ -1,29 +1,28 @@
 --find-links=https://packages.dea.ga.gov.au/hdstats/
-aiohttp
-awscli
-botocore
+aiohttp==3.10.10
+awscli==1.38.1
+botocore==1.37.1
 Bottleneck==1.4.2
 click==8.1.7
 dask==2024.3.1
 datacube[s3,performance]==1.8.19
-dea-tools>=0.3.4
+dea-tools==0.3.6
 eodatasets3==0.30.6
 eo-tides==0.6.3
 hdstats==0.1.8.post1
 geopandas==0.14.4
 matplotlib==3.9.2
-mdutils
+mdutils==1.6.0
 numpy==1.26.4
 odc-algo==0.2.3
 odc-geo==0.4.8
-odc-ui
+odc-ui==0.2.1
 pandas==2.2.3
 pyogrio==0.10.0
 pyproj==3.7.0
 pytest==8.3.3
 pytest-dependency
 pytest-cov
-pyTMD>=2.2.1
```

Comment:
Removing pyTMD here to allow it to be controlled by `eo-tides`'s own dependencies